bulk-load/src/main/java/no/sikt/nva/data/report/api/etl/loader/LoaderSpec.java

```diff
@@ -0,0 +1,98 @@
+package no.sikt.nva.data.report.api.etl.loader;
+
+import java.net.URI;
+import java.util.Locale;
+
+public record LoaderSpec(URI source,
+                         Format format,
+                         String iamRoleArn,
+                         String region,
+                         boolean failOnError,
+                         Parallelism parallelism,
+                         boolean updateSingleCardinalityProperties,
+                         boolean queueRequest) {
+
+    @Override
+    public String toString() {
```

Comment:
This isn't pretty, can consider a JSON workflow later

Reply:
you can implement json serialize and override the toSting method to use the toJsonString.